src/Servers/Kestrel/Core/ref/Microsoft.AspNetCore.Server.Kestrel.Core.netcoreapp.cs

```diff
@@ -74,6 +74,8 @@ public partial class Http2Limits
         public int HeaderTableSize { get { throw null; } set { } }
         public int InitialConnectionWindowSize { get { throw null; } set { } }
         public int InitialStreamWindowSize { get { throw null; } set { } }
+        public System.TimeSpan? KeepAlivePingInterval { get { throw null; } set { } }
+        public System.TimeSpan? KeepAlivePingTimeout { get { throw null; } set { } }
```

Comment:
> FYI KestrelServerLimits uses non-nullable TimeSpans with Timeout.InfiniteTimeSpan.

Since you mentioned this yourself, I though you were going to change this to regular TimeSpan. Kestrel doesn't use `TimeSpan?` anywhere else.

Reply:
I brought it up as something to think about.

Do you want the ideal (TimeSpan?) or do you want consistency? (TimeSpan + Timeout.InfiniteTimeSpan)

Reply:
I like consistency.